pkg/storaged/stratis/pool.jsx

```diff
@@ -110,8 +110,8 @@ function create_fs(pool) {
                                                           variant == "nomount");
                           }
                       }),
-            mount_options(false, false),
-            at_boot_input(client.in_anaconda_mode() ? "local" : "nofail"),
+            mount_options(false, false, () => !client.in_anaconda_mode()),
+            at_boot_input(client.in_anaconda_mode() ? "local" : "nofail", () => !client.in_anaconda_mode()),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19827-20240130-100343-b78ee803-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)